docker-compose.yml

```diff
@@ -1,8 +1,27 @@
 version: '3.4'
 
+x-shared-variables: &shared-variables
+  DATABASE_URL: postgres://postgres:password@db/submitter_local
+  NOTIFY_EMAIL_GENERIC: 46a72b64-9541-4000-91a7-fa8a3fa10bf9
+  NOTIFY_EMAIL_RETURN_SETUP_EMAIL_TOKEN: 38f6a1cd-a810-4f59-8899-2c300236c5b4
+  NOTIFY_EMAIL_RETURN_SETUP_EMAIL_VERIFIED: ''
+  NOTIFY_EMAIL_RETURN_SETUP_MOBILE_VERIFIED: 54dcaad7-4967-431d-8606-72b0b80b5c6a
+  NOTIFY_EMAIL_RETURN_SIGNIN_EMAIL: ''
+  NOTIFY_EMAIL_RETURN_SIGNIN_SUCCESS: ''
+  NOTIFY_SMS_GENERIC: aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa
+  NOTIFY_SMS_RETURN_SIGNIN_MOBILE: something
+  NOTIFY_SMS_RETURN_SETUP_MOBILE: 54dcaad7-4967-431d-8606-72b0b80b5c6a
+  FB_ENVIRONMENT_SLUG: test
+  SECRET_KEY_BASE: xxxyyy
+  SERVICE_TOKEN_CACHE_ROOT_URL: http://fake_service_token_cache_root_url/
+  PDF_GENERATOR_ROOT_URL: http://pdf-generator.com
+  ENCRYPTION_KEY: i6USnzeRKljfLMPbRlB2E9oikURx4ou3
```

Comment:
Just to check these are random nonesense values you generated for this source version?